src/main/java/parking/repositories/LogRepositoryImpl.java

```diff
@@ -19,11 +18,11 @@ public LogRepositoryImpl(MongoOperations operations) {
         this.operations = operations;
     }
 
-    public void insertActionLog(ActionType actionType, ObjectId targetUserId, Integer lotNumber, Date from, Date to, LogMetaData metaData, ObjectId userId, String channel) {
+    public void insertActionLog(ActionType actionType, Account targetUser, Integer lotNumber, Date from, Date to, LogMetaData metaData, Account user, String channel) {
```

Comment:
I mean userAgent by channel, perhaps I should change the name to userAgent if it's more clear to most people.